airbyte-integrations/connectors/source-mssql/src/test-integration/java/io/airbyte/integrations/source/mssql/CdcMssqlSourceAcceptanceTest.java

```diff
@@ -178,4 +173,17 @@ private List<AirbyteStateMessage> filterStateMessages(final List<AirbyteMessage>
         .collect(Collectors.toList());
   }
 
+  @Test
+  @Disabled
```

Comment:
Can you add a string explaining why it's disabled, other this might more easily end up being disabled for ever.

Reply:
Same comment applies below.